src/command/backup/backup.c

```diff
@@ -1148,15 +1149,15 @@ backupJobResult(
             PackRead *const checksumPageResult = pckReadPackReadP(jobResult);
 
             // Increment backup copy progress
-            sizeCopied += copySize;
+            *sizeProgress += copySize;
 
             // Create log file name
             const String *fileLog = host == NULL ? fileName : strNewFmt("%s:%s", strZ(host), strZ(fileName));
 
             // Format log strings
             const String *const logProgress =
                 strNewFmt(
-                    "%s, %" PRIu64 "%%", strZ(strSizeFormat(copySize)), sizeTotal == 0 ? 100 : sizeCopied * 100 / sizeTotal);
+                    "%s, %" PRIu64 "%%", strZ(strSizeFormat(copySize)), sizeTotal == 0 ? 100 : (*sizeProgress) * 100 / sizeTotal);
```

Comment:
I don't think parens are needed here. We generally add parens only to establish precedence or if the compiler gripes.

Reply:
will remove